src/internet/subsonic/subsonicdynamicplaylist.cpp

```diff
@@ -40,13 +40,16 @@ const int SubsonicDynamicPlaylist::kDefaultSongCount = 20;
 const int SubsonicDynamicPlaylist::kDefaultOffset = 0;
 
 SubsonicDynamicPlaylist::SubsonicDynamicPlaylist()
-  : type_(QueryType_Album), stat_(QueryStat_Newest), offset_(kDefaultOffset) {}
+  : type_(QueryType_Album), stat_(QueryStat_Newest), offset_(kDefaultOffset) {
+  service_ = InternetModel::Service<SubsonicService>();
+}
 
 SubsonicDynamicPlaylist::SubsonicDynamicPlaylist(const QString& name,
                                                  QueryType type,
                                                  QueryStat stat)
     : type_(type), stat_(stat), offset_(kDefaultOffset) {
   set_name(name);
+  service_ = InternetModel::Service<SubsonicService>();
```

Comment:
Probably doesn't make a difference, but the service could be passed in to this constructor.

Reply:
I'd rather keep it like that for consistency with the other constructor that has to be parameterless.

Reply:
I'd rather keep it like that for consistency with the other constructor that has to be parameterless.